### PR TITLE
fix(transition guide): edited the transition guide

### PR DESCRIPTION
**Pull request by homelessbirds**

### CODE REVIEW

**Review by jeff-colucci — DISMISSED** [review]
No issues to my eyes: looks good!

File: src/content/docs/new-relic-solutions/new-relic-one/new-navigation-transition-guide.mdx [hunk 1]
```diff
@@ -19,61 +19,91 @@ import transitionGuideUserMenu from 'images/transition-guide_screenshot-full_use
...
+  alt="A gif demonstrating how to use the query builder in the New Relic UI. "
+  src={transitionGuideQueryBuilderUiRedesign}
 />
 
-With this update you have a simpler, more stable UI experience that gets out of your way and creates more screen real estate so you can focus on what's most important: the health of your system.
+<figcaption>
+  Pin the query builder to the left navigation to quickly access your data. 
+</figcaption>
+
+Before you dig into making New Relic your own, the first thing we recommend is that you go to **All capabilities**  and click the pin next to "Query your data." This will permanently stick the query builder to the left navigation.  Our new [query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/) is an essential piece of the New Relic ecosystem that allows you an easy way to filter, organize, and understand your data.
```
Comment (zuluecho9) [inline src/content/docs/new-relic-solutions/new-relic-one/new-navigation-transition-guide.mdx:73]: note that it's not 'new'. prob just take out 'new'.

Comment (zuluecho9) [inline src/content/docs/new-relic-solutions/new-relic-one/new-navigation-transition-guide.mdx:73]: probably could just delete that last sentence going 'is an essential piece' and link from 'query builder' to its doc at first mention. we explain below what it does so I wouldn't belabor the point too much; most existing customers know what the query builder is.

File: src/content/docs/new-relic-solutions/new-relic-one/new-navigation-transition-guide.mdx [hunk 2]
```diff
@@ -19,61 +19,91 @@ import transitionGuideUserMenu from 'images/transition-guide_screenshot-full_use
...
-## Browse for any technology you use [#add-more]
+### Metrics and events   [#metrics-and-events]
+
+<img
+  title="Metrics and events"
+  alt="A gif demonstrating the New Relic redesign's metrics and events."
+  src={transitionGuideMetricsandEvents}
+/>
+
+Our second recommendation is that you go to **All capabilities**  and pin "Metrics and events" to the left navigation. Formerly included in the UI as the "data explorer," [metrics and events](docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer) allows your team a birds-eye view of all the data coming into your system. You can use metrics and events to fetch, identify and understand all your data stored in the New Relic database (NRDB) through visual tools. You can explore your data without having to run any queries making it one of the simplest tools for gathering information about the health of your system.
```
Comment (zuluecho9) [inline src/content/docs/new-relic-solutions/new-relic-one/new-navigation-transition-guide.mdx:90]:
```suggestion
Our second recommendation is that you go to **All capabilities**  and pin "Metrics and events" to the left navigation. Formerly included in the UI as the "data explorer," [metrics and events](/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer) allows your team a birds-eye view of all the data coming into your system. You can use metrics and events to fetch, identify and understand all your data stored in the New Relic database (NRDB) through visual tools. You can explore your data without having to run any queries making it one of the simplest tools for gathering information about the health of your system. 
```

File: src/content/docs/new-relic-solutions/new-relic-one/new-navigation-transition-guide.mdx [hunk 3]
```diff
@@ -167,13 +198,12 @@ Go to **All entities** to see all of your monitored entities in one place.
...
+### We've moved the user menu [#user-menu]
 
 <img
   title="The new user menu"
   alt="A screenshot of the new user menu location in the bottom left corner."
   src={transitionGuideUserMenu}
-  style={{ maxWidth: '70%' }}
 />
 
 Find your user menu in the bottom left corner of the UI. Use it to invite a teammate to New Relic, set your user preferences, switch to dark mode, find or create API keys, and change other settings.
```
Comment (zuluecho9) [inline src/content/docs/new-relic-solutions/new-relic-one/new-navigation-transition-guide.mdx:209]:
```suggestion
Find your user menu in the bottom left corner of the UI. Use it to invite a teammate to New Relic, set your user preferences, switch to dark mode, find or create API keys, and change other settings. To find details about these options, see [Account settings](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings).
```

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 4]
```diff
@@ -86,6 +86,61 @@ Decisions determine how incident intelligence correlates issues together. New Re
...
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="configure-source-nr-alerts"
+    title="Alerts"
+  >
+    By integrating incident intelligence with your alerts violations, you can get context and correlations from what you're monitoring.
+    To get data from alerts:
+
+    1. From **[one.newrelic.com](https://one.newrelic.com)**, click **Alerts**.
```
Comment (akristen) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:104]: For fast follows: link should be **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 5]
```diff
@@ -86,6 +86,61 @@ Decisions determine how incident intelligence correlates issues together. New Re
...
+    2. On the left under **incident intelligence**, click **Sources** and then click **Alerts**.
```
Comment (akristen) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:105]: Should Incident have a capital I? (I can't see it in my one.nr.com to x2 check)

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 6]
```diff
@@ -86,6 +86,61 @@ Decisions determine how incident intelligence correlates issues together. New Re
...
+         Adding alerts as a source will not affect your current configuration or notifications.
+       </Callout>
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="configure-aporia"
+    title="Aporia (MLOps)"
+  >
+    By integrating incident intelligence with your Aporia machine-learning models, you can monitor your machine learning model performance. To configure our Aporia integration, see our [docs](/docs/integrations/mlops-integrations/aporia-mlops-integration/).
```
Comment (akristen) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:120]: For see our [docs], I'd have a longer phrase besides 'docs' so users know roughly what kind of doc it is

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 4 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:104]: done in a fast follow! thanks!

**Pull request merged**